### DISCUSSION TITLE
Any idea how to update the latest version in VSCodium / open-vsx.org?

### DISCUSSION THREAD

**Discussion by metaleap** [mike-lischke/vscode-antlr4#287] [Q&A] [answered]
While `vscode-antlr4` is at [2.4.6](https://marketplace.visualstudio.com/items?itemName=mike-lischke.vscode-antlr4), the OpenVSX version is at [2.3.1](https://open-vsx.org/extension/mike-lischke/vscode-antlr4) — what I don't know is whether @mike-lischke is the maintainer of the latter — or whether some community anon put it up there back when 2.3.1 was current, but is perhaps no longer on it (wrt pushing updates there).

Anyone know the usual workflow to update this with OpenVSX? Haven't found a proper action button or something for that there. Only to "publish" one's own stuff from scratch. (That they don't update exts automagically when both their Github repo and MS marketplace entry are more recent — is quite a bother..  =)

**Comment by mike-lischke** [comment, marked as answer]
I was asked to publish on open-vsx.org as well, but since I don't use VS Codium, I declined. So someone else must have done it there.

**Reply by metaleap** [reply]
Fair enough, I got every bit as far as I wanted to even with the old version; thanks for your work  =)
